src/config/color-schemes.drconf

```diff
@@ -10,6 +10,18 @@ text: #e8e6e3
 
 ================================
 
+One Dark
```

Comment:
Themes must be added in alphabetical order to pass tests before merge. Can you fix that?

Reply:
Hi, firstly, thanks for your response.

Sure, I changed the commit to order the file.